src/sagemaker_xgboost_container/constants/xgb_constants.py

```diff
@@ -10,6 +10,31 @@
 # distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
+
+XGB_MAXIMIZE_METRICS = [
```

Comment:
Some of these are actually mis-categorized. We should probably fix them in this PR.

According to master: https://github.com/aws/sagemaker-xgboost-container/blob/master/src/sagemaker_xgboost_container/constants/xgb_constants.py#L14

Reply:
+1 - worthwhile pulling https://github.com/aws/sagemaker-xgboost-container/commit/cd4fb0d86a608c050e714c9125b01847f4bc4587 only if it's low cost and only if it's already in 0.90-2 (otherwise it will become a regression).

Reply:
It actually isn't in 0.90-2. It looks like that commit was only applied from 1.0-1 onward. Should we leave it as-is then, or also cherry-pick it into 0.90-2?

Reply:
I suggest we leave it for now since we don't want to delay this fix.